examples/reference_environments/snowflake/compose.yaml

```diff
@@ -0,0 +1,11 @@
+services:
+  jupyter:
+    image: gx/snowflake_example_jupyter
+    container_name: gx_snowflake_example_jupyter
+    build:
+      context: .
+      dockerfile: jupyter.Dockerfile
+    environment:
+      CONNECTION_STRING: $(SNOWFLAKE_CONNECTION_STRING)
```

Comment:
Using connection string rather than having each "component" (ie. user name, password, schema.. ) be their own component be their own env variable